docs/migrate-from-masb/mssqlfg.md

```diff
@@ -0,0 +1,81 @@
+# Migrate [Azure SQL Database Failover Group Service](https://github.com/Azure/meta-azure-service-broker/blob/master/docs/azure-sql-db-failover-group.md) Instances From MASB To OSBA
+
+## Steps
+
+### Create azure-sql-12-0-dr-dbms-pair-registered service instance by OSBA
+
+OSBA doesn't support to provide existing servers' credentials from manifest and create database on them. You need to create a **azure-sql-12-0-dr-dbms-pair-registered** service instance for the two servers involved:
+
+```
+cf create-service azure-sql-12-0-dr-dbms-pair-registered dbms <sql-server-pair-instance-name> -c '{
+  "primaryResourceGroup":"<primary-group-name>",
+  "primaryServer":"<primary-server-name>",
+  "primaryLocation":"<primary-server-location>",
+  "primaryAdministratorLogin":"<primary-login>",
+  "primaryAdministratorLoginPassword":"<primary-login-password>",
+  "secondaryResourceGroup":"<secondary-group-name>",
+  "secondaryServer":"<secondary-server-name>",
+  "secondaryLocation":"<secondary-server-location>",
+  "secondaryAdministratorLogin":"<secondary-login>",
+  "secondaryAdministratorLoginPassword":"<secondary-login-password>",
+  "alias":"<sql-server-pair-instance-name>"
+}'
+```
+
+### Create azure-sql-12-0-dr-database-pair-from-existing service instance by OSBA
+
+In MASB, the service for failover group creates the secondary database and failover group based on an existing primary database. In OSBA, the primary database, the secondary primary, and the failover group are managed by one service instance. To migrate, you need to create **azure-sql-12-0-dr-database-pair-from-existing** service instance.
```

Comment:
Should be "the secondary database"

Reply:
Thanks for catching it... will fix the typo.